src/commands.ts

```diff
@@ -50,7 +50,7 @@ export async function setupBunfigToml(dir: string) {
   const msg = `Setting up ${BUNFIG_FILE}`;
   try {
     let content = await fs.promises.readFile(bunfigPath, "utf-8");
-    if (!/^"@myorg1"\s+=/gm.test(content)) {
+    if (!/^"@myorg1"\s+=/gm.test(content) && !content.includes(JSR_BUNFIG)) {
```

Comment:
Let's only check if a `@jsr` registry mapping is in there.

```suggestion
   if (!/^"@jsr"\s+=/gm.test(content)) {
```

Reply:
Thank you for reviewing! I see. I fixed it!